components/pedigree/resources/src/main/resources/pedigree/pedigree.js

```diff
@@ -111,6 +113,14 @@ define([
 
             this._preferencesManager.load( function() {
                     Helpers.copyProperties(PedigreeEditorParameters.styles.blackAndWhite, PedigreeEditorParameters.attributes);
+                    // set up constants which depend on preferences
+                    if (editor.getPreferencesManager().getConfigurationOption("lineStyle") == "thin") {
+                        Helpers.copyProperties(PedigreeEditorParameters.lineStyles.thinLines, PedigreeEditorParameters.attributes);
```

Comment:
Better use `var lineStyleConfig = editor.getPreferencesManager().getConfigurationOption("lineStyle");` and may be also do a `switch (lineStyleConfig)` statement.